Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -1159,6 +1160,10 @@ extension NavigatorIndex {
             return problems
         }
         
+        public func getAvailabilityEntryIDs(for availabilityID: UInt64) -> [Int]? {
```

Comment:
Maybe this should be internal for now? Otherwise, if you think it should be public, we should document it.